azurerm/resource_arm_redis_cache_test.go

```diff
@@ -578,11 +597,40 @@ resource "azurerm_redis_cache" "test" {
       maxmemory_delta    = 2
       maxmemory_policy   = "allkeys-lru"
     }
-
     patch_schedule {
       day_of_week    = "Tuesday"
       start_hour_utc = 8
     }
 }
 `, rInt, location, rInt)
 }
+
+func testAccAzureRMRedisCacheSubscribeAllEvents(rInt int, rString string, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+    name = "acctestRG-%d"
+    location = "%s"
+}
+resource "azurerm_storage_account" "test" {
+  name                = "unlikely23exst2acct%s"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  location     = "${azurerm_resource_group.test.location}"
+  account_type = "Standard_GRS"
```

Comment:
in running the tests I noticed this field needs to be split into the two sub-fields - I'll push a commit to fix this :)

```
account_tier             = "Standard"
account_replication_type = "GRS"
```

Reply:
Oh...thanks for your help :D